home/pi/auto_run.sh

```diff
@@ -548,6 +566,85 @@ then
     rm ~/first_run
 fi
 
+# Matrix Voice Hat Setup
+if [ -f ~/setup_matrix ]
+then
+    if [ ! -f matrix_setup_state.txt ]
+    then
+        echo ""
+        echo "========================================================================="
+        echo "Setting up Matrix Voice Hat. This will install the matrixio-kernel-modules and pulseaudio"
+        echo "This will require 3 reboots"
```

Comment:
echo "This process is automatic, but requires rebooting three times.  Please be patient!"